src/promptflow/promptflow/_sdk/entities/_eager_flow.py

```diff
@@ -28,6 +28,12 @@ def __init__(
         code = Path(code)
         # entry function name
         self.entry = entry
+        # entry file name
+        try:
+            # TODO: remove this after gen meta removed entry file
+            self.entry_file = f'{entry.split(":")[0].replace(".", "/")}.py'
```

Comment:
do we need to resolve this to absolute path somewhere?

Reply:
Executor can resolve it according to relative path and working directory. I guess we can resolve it to absolute early.